frontend/src/app.vue

```diff
@@ -6,65 +6,69 @@
     v-bind:opacity='loadingOverlayOpacity',
     v-bind:is-full-page="true"
   )
-    i.overlay-loading-icon.fa.fa-spinner.fa-spin(slot='default')
-    h3(slot='after') {{ loadingOverlayMessage }}
+    template(slot:default)
+      i.overlay-loading-icon.fa.fa-spinner.fa-spin()
+    template(slot:after)
```

Comment:
@gerhean Can I double check if `slot` should be changed to `v-slot` here?

Reply:
It's the same, but I suppose  `v-slot` is safer.